clients/ctl/admin-ui/src/features/dataset/DatasetFieldsTable.tsx

```diff
@@ -20,12 +21,15 @@ import { ColumnMetadata, EditableType } from "./types";
 const Cell = ({
```

Comment:
This component is getting big enough I would extract it its own file. However, Chakra is big picky about what can go under a Td element, and I'm worried that coupling would be unclear if this were in its own file. 😢

Reply:
I think it'd probably be okay as its own file as long as there's a comment about the Td pickiness

Reply:
Put that change into my [renames PR](https://github.com/ethyca/fides/pull/1113/commits/d1d7f39dcb6747deeeb1e456664fbc42419be676).